scout/server/blueprints/variants/controllers.py

```diff
@@ -462,6 +463,19 @@ def swegen_link(variant_obj):
                     "{this[position]}-{this[reference]}-{this[alternative]}")
     return url_template.format(this=variant_obj)
 
+def cosmic_link(variant_obj):
+    """Compose link to COSMIC Database."""
```

Comment:
Could you create a proper docstring here? I know it is probably lacking on the other functions here but let's work together and improve all code. 
It should look something like:
```python
def cosmic_link(variant_obj):
    """Compose link to COSMIC Database.

    Args:
        variant_obj(scout.models.Variant)

    Returns:
        url_template(str): Link to COSMIIC database if cosmic id is present
    """
```